geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/command/Size.java

```diff
@@ -72,7 +72,7 @@ public void cmdExecute(final @NotNull Message clientMessage,
     String regionName = regionNamePart.getCachedString();
 
     if (regionName == null) {
-      logger.warn("The input region name for the %s request is null", "size");
+      logger.warn("The input region name for the size request is null");
```

Comment:
while you are in this part of the code consider fixing the errMessage.append call also. It seems like we should just have a local String constant that is in both calls.

Reply:
Done with https://github.com/apache/geode/pull/7399